backend/server/src/main.rs

```diff
@@ -10,6 +10,7 @@ mod web;
 use anyhow::Context;
 use client::UnixService;
 use config::Config;
+use std::sync::mpsc::channel;
```

Comment:
```suggestion
use tokio::sync::mpsc::channel;
```

If we use std channels, we end up blocking in one of tokio's worker threads, which if done to often, will bring the whole runtime to a halt. If we really needed to use std channels (or other blocking channels) we could use tokio's `spawn_blocking`, but we do not (afaik), so we should just use async channels.

Reply:
Sounds good, done